scripts/e2e/config/kserve/overlay.yaml

```diff
@@ -8,12 +8,13 @@ spec:
     spec:
       containers:
       - name: manager
+        image: kserve/kserve-controller:latest
         resources:
           limits:
-            cpu: 100m
+            cpu: 1
```

Comment:
It's quite big ya. What if we don't specify the CPU limits?

Reply:
Just in case. Gitlab runner has 2 CPU assigned to each job.